src/main/resources/assets/chisel/lang/de_de.lang

```diff
@@ -0,0 +1,2774 @@
+item.chisel.chisel_iron.name=Meißel
+item.chisel.chisel_diamond.name=Diamantmeißel
+item.chisel.chisel_hitech.name=iMeißel
+item.chisel.offsettool.name=Ender-Versatzstab
+
+chisel.spawnwither=Ein Meißel kann etwas gefährliches sein...
+
+item.chisel.chisel.desc.gui=Rechtsklick um das GUI zu öffnen
+item.chisel.chisel.desc.lc1=Linksklick um Blöcke in der Welt zu meißeln
+item.chisel.chisel.desc.lc2=Wähle einen Block indem du ihn im Inventar ablegst
+
+item.chisel.chisel.desc.modes=Hat mehrere Meißelmodi.
+
+container.chisel.title=Meißel
+
+itemGroup.chiselCreativeTab=Meißel
+chisel.jei.title=Meißeln
+
+jei.chisel.desc.concrete_making=Füge nur etwas Wasser hinzu und schon kannst du deinen eigenen Beton herstellen!\n\nTauche Betonpulver in Wasser indem du es daneben plazierst oder lass' es ins Wasser fallen.
+
+chisel.tooltip.blockname=%1$s (%2$s)
+
+tile.chisel.blockAluminum.name=Aluminumblock
+tile.chisel.blockAluminum.caution.desc.1=Vorsicht
+tile.chisel.blockAluminum.crate.desc.1=Versandkiste
+tile.chisel.blockAluminum.thermal.desc.1=Warm
```

Comment:
the thermal block in-game look like a heated block. I think something like "Warm" or "Erwärmt" or "Glühend" would be matching but "Thermal" sounds .. wrong in german.
I would like to have more ideas for that. What would be your translation?